.github/workflows/release-vm-console-proxy.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release VM Console Proxy
+
+on:
+  repository_dispatch:
+    types: [release-vm-console-proxy]
+
+jobs:
+  release-vm-console-proxy:
+    name: Release VM Console Proxy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          release_version: ${{ github.event.client_payload.release_version }}
+      - run: |
+          RELEASE_VERSION=${{ github.event.client_payload.release_version }}
+          OUTPUT_FILE=./data/vm-console-proxy-bundle/vm-console-proxy-${RELEASE_VERSION}.yaml
```

Comment:
No need to add release version to the filename. There will be only one file.

Reply:
Okay, but it's similar to common-templates [1] and kubevirt-tekton-tasks-operator [2].

[1] https://github.com/kubevirt/ssp-operator/tree/master/data/common-templates-bundle
[2] https://github.com/kubevirt/tekton-tasks-operator/tree/main/data/tekton-tasks/kubernetes

Reply:
FWIW we wanted to move away from including all of the versioned manifests in the operator, that's why `common-instancetypes` just has one now.

Reply:
Ack. I'll remove it now.